wg-deployment/README.md

```diff
@@ -8,13 +8,16 @@ To understand how this file is generated, see https://github.com/kubeflow/commun
 --->
 # Deployment Working Group
 
+<hr>
+<h3>⚠️This Working Group is INACTIVE ⚠️</h3>
```

Comment:
I think, we should follow the process defined here: https://github.com/kubeflow/community/blob/3821b98e2c2036e09fcd985ccf8e35ebd3fc89ce/wgs/wg-lifecycle.md#steps
@thesuperzapper Can you move in-active SIGs and WGs to the `archive` sub-directory, and announce it in the `kubeflow-discuss`.